packages/components/src/components.ts

```diff
@@ -52,7 +62,7 @@ export { default as HdsButton } from './components/hds/button/index.ts';
 export { default as HdsButtonSet } from './components/hds/button-set/index.ts';
 
 // Card
-export { default as HdsCard } from './components/hds/card/container.ts';
+export { default as HdsCardContainer } from './components/hds/card/container.ts';
```

Comment:
I though we have renamed this component to just be HdsCard a while back? I might be mistaken tho

Reply:
No, always been `CardContainer`, in case one day we introduce a generic `Card` component (similar to what is happening for the `StepperStepIndicator/StepperTaskIndicator` elements)

Reply:
Oh ok, I guess we missed it when we introduced component.ts reexport file. I have some fixing to do on the next upgrade in cloud-ui 🤣